eZ/Publish/Core/settings/search_engines/legacy/criterion_handlers_common.yml

```diff
@@ -33,10 +33,10 @@ parameters:
         stopWordThresholdFactor: 0.66
         enableWildcards: true
         commands:
+            - "ascii_search_cleanup"
             - "apostrophe_normalize"
             - "apostrophe_to_doublequote"
             - "ascii_lowercase"
-            - "ascii_search_cleanup"
```

Comment:
On ´it's´ => ´it´ & ´s´ I don't think we need to emulate legacy, aka I don't think it should be indexed as two words. So with that out of the way, does that mean we can remove this change and not?  aka what else was this part of the change for?